src/main/java/stsjorbsmod/patches/NlothsGiftPatch.java

```diff
@@ -0,0 +1,81 @@
+package stsjorbsmod.patches;
+
+import com.evacipated.cardcrawl.modthespire.lib.*;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.events.shrines.Nloth;
+import com.megacrit.cardcrawl.relics.AbstractRelic;
+import javassist.CannotCompileException;
+import javassist.CtBehavior;
+import javassist.expr.ExprEditor;
+import javassist.expr.FieldAccess;
+import stsjorbsmod.relics.FragileMindRelic;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+
+public class NlothsGiftPatch {
+
+    /**
+     * Removes FragileMindRelic if found in the iterator
+     *
+     * @param iter
+     */
+    public static void removeFragileMindRelic(Iterator<AbstractRelic> iter) {
+        while (iter.hasNext()) {
+            if (FragileMindRelic.ID.equals(iter.next().relicId)) {
+                iter.remove();
+            }
+        }
+    }
+
+    /**
+     * This is called only by edited expressions in the main game. See the following SpirePatch.
+     *
+     * @param playerRelics
+     * @return
+     */
+    public static List<AbstractRelic> clonePlayerRelicsWithoutFragileMind(List<AbstractRelic> playerRelics) {
+        List<AbstractRelic> relics = new ArrayList<>(playerRelics);
+        removeFragileMindRelic(relics.iterator());
+        return relics;
+    }
+
+    @SpirePatch(clz = AbstractDungeon.class, method = "getShrine")
+    public static class AbstractDungeon_getShrine_NlothCheck {
+        public static ExprEditor Instrument() {
+            return new ExprEditor() {
+                @Override
+                public void edit(FieldAccess fieldAccess) throws CannotCompileException {
+                    if (fieldAccess.getClassName().equals(AbstractPlayer.class.getName())
+                            && fieldAccess.getFieldName().equals("relic")) {
+                        fieldAccess.replace("{ $_ = " + NlothsGiftPatch.class.getName() + ".clonePlayerRelicsWithoutFragileMind($0)); }");
```

Comment:
`$0` isn't what you want here; when you're doing a field access patch for original code `someObject.someField`, `$0` represents `someObject`. What you're going to want to do is going to look almost identical to the `CloneMasterDeckWithoutLegendaryCardsEditor` class in `LegendaryPatch`; I'd recommend taking a look at what that's doing :)